### PR TITLE
Project Proposal Submission

### PR DESCRIPTION
**Pull request by Traian-Pop**
Traian Pop
43560768

### CODE REVIEW

File: proposal.md [hunk 1]
```diff
@@ -1,20 +1,9 @@
...
 ## Outline Structure
 
-> Describe how you'll organize your code. What is the process and
-> supervision structure? If it uses a framework, how does it fit in. I
-> just need to understand the direction you plan to take. Again, three
-> or four sentences are probably enough.
-
-
+My project will first of all have a human interface for the player to interact with the "casino". Then, there will be two modules: one in charge of the deck/numbers and the other for calculating all the percentages involved in the player's decision. Lastly, there will be a supervised server in charge of all the decisions involved in this process.
```
Comment (pragdave) [inline proposal.md:8]: I'm not clear what the supervisor is managing here. What servers will be running.

**Comment by pragdave** [comment]
Please clarify my question about servers before continuing. Update the PR as soon as possible.

File: proposal.md (hunk 1 above)
Comment (Traian-Pop) [reply proposal.md:8]: I just realized I didn't really explain a part of my project very well: one of my goals is to have a supervised "casino" server where multiple people can join in and play games, similar to how we did Assignment 4. The children of the casino supervisor would manage all the separate games/tables.

**Comment by pragdave** [comment]
# Approved

**Pull request closed without merging**